### ISSUE TITLE
“now” date is UTC, not local

### ISSUE THREAD

**Issue by ticky** [cobalt-org/liquid-rust#332]
The “now” keyword in Shopify Liquid calls out to [Ruby's `Time.now`,](https://github.com/Shopify/liquid/blob/8dcc3191281478c4ba544d3c507fdb99aa512f75/lib/liquid/utils.rb#L72-L73) which returns the current time, in the current system time zone:

```
$ irb
irb(main):001:0> Time.now
=> 2019-02-08 15:34:25 -0800
```

liquid-rust’s implementation uses [the current time in UTC](https://github.com/cobalt-org/liquid-rust/blob/2b01636c754a70248b1ee61b43b8369ba502414c/liquid-value/src/scalar.rs#L387-L392), which makes templates relying on the upstream Liquid behaviour behave incorrectly!

For instance, my site uses `{{ 'now' | date: "%b %-d, %I:%M %P %Z" }}` to print the compilation time in the footer of pages, and it is my intent that this shows in the local time zone of the machine that generated it, however, this now emits the date in UTC, which is less desirable to me.

---

liquid-rust version: master
rust version: 1.32.0
OS: macOS 10.14

**Comment by epage** [comment]
It does? Thats disappointing. If you use a CI to build your site, it is unpredictable what time zone it will be in.

Oh, well. The joy of compatibility.

**Comment by ticky** [comment]
It’s definitely not a great stance, and I think the suggested workaround is to adjust the time zone of the application (i.e. you could do this by setting `TZ`, or as Jekyll does, by adjusting it at the build config level), but given that Liquid makes no attempt to allow you to _select_ a time zone for display otherwise, it’d be good to keep it consistent :)

**Comment by epage** [comment]
Agreed though I think it might be nice to offer a feature flag to change from local to UTC.
